src/core.js

```diff
@@ -848,7 +848,7 @@ $.extend( $.validator, {
 			// aria-describedby should directly reference the error element
 			if ( describer ) {
 				selector = selector + ", #" + describer
-					.replace( /'/g, "\\'" )
+					.replace( /('|:|\.|\[|\]|\$)/g, "\\$1" )
```

Comment:
I meant line https://github.com/jzaefferer/jquery-validation/pull/1614/files#diff-7eb52b366866677666470e019283c8eaR814

We can just use a local var, dont need a global one.